kustomize/kustomize.go

```diff
@@ -300,3 +305,20 @@ func GetUnstructuredObjects(data []byte) ([]runtime.Object, error) {
 
 	return items, nil
 }
+
+func BytesToUtf8Lf(file []byte) (string, error) {
+	decoded := utfutil.BytesReader(file, utfutil.UTF8)
+	buf := new(bytes.Buffer)
+	bytesRead, err := buf.ReadFrom(decoded)
+	if err != nil {
+		logger.Errorf("error reading from buffer. bytesRead %v. err: %v", bytesRead, err)
+		return "", err
+	}
+	val := buf.Bytes()
+	// replace \r with \n -> solves for Mac but leaves \n\n for Windows
+	val = bytes.Replace(val, []byte{13}, []byte{10}, -1)
```

Comment:
```suggestion
	val = bytes.ReplaceAll(val, []byte{13}, []byte{10})
```

Reply:
^since you want to replace all the occurrence ReplaceAll should be better choice here

Reply:
Addressed